localisation/00_zuwc_united_localisation.csv

```diff
@@ -2920,12 +2920,13 @@ EVTOPTA_WCPRM_600;As water wish.;;Wie Wasser w
 EVTOPTB_WCPRM_600;No! You have completely lost your hearing, old man!;;Nein! Ihr habt vollkommen das Geh�r verloren, Alter!;;;;;;;;;;;x
 EVTDESC_WCPRM_605;The oracle frowns and cries: "I'm the voice of the water, young one. The rivers want to warn you. If you decide to remain deaf and go against the tide, well, it is not in my power to convince you.";;Das Orakel runzelt die Stirn und schreit: �Ich bin die Stimme des Wassers, J�ngling. Die Fl�sse wollen dich warnen. Wenn du dich entscheidest, taub zu bleiben und gegen den Strom zu schwimmen, nun, dann liegt es nicht in meiner Macht, dich zu �berzeugen.�;;;;;;;;;;;x
 EVTOPTA_WCPRM_605;Old fool!;;Alter Narr!;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+pick_titanic_branch;Choose Titanic Way;;;;;;;;;;;;;x
+pick_titanic_branch_desc;Choose the way that you will follow in the name of the creators.;;;;;;;;;;;;;x
```

Comment:
```suggestion
pick_titanic_branch_desc;Choose the path to follow in honoring the creators.;;;;;;;;;;;;;x
```

Reply:
Done in 0aab3031ea13c6880134053c2590a6aef6984bde.